general_autoencoder.py

```diff
@@ -230,7 +281,14 @@ def _fit_from_processed_data(self, train_data, valid_data, train_ages=None, vali
         
         print("Train size %i; valid size %i" % (
             self.train_data.shape[0], self.valid_data.shape[0]))
-                
+        
+        if self.uses_longitudinal_data:
```

Comment:
we store all the other data a couple lines above; this seemed consistent.

Reply:
Ok, I guess the lines above might not be in the right place then. Probably not a big deal (and this might well be my fault in the original code) but I don't understand when we pass data around and when we keep it in the object. Seems good to either always pass data around, or put it in the object at the top level?

Reply:
yeah perhaps so. this feels peripheral to this PR so i will put it as an issue but leave as is for now.